go/vt/vtgate/buffer/flags.go

```diff
@@ -70,9 +70,6 @@ func verifyFlags() error {
 	if bufferSize < 1 {
 		return fmt.Errorf("--buffer_size must be >= 1 (specified value: %d)", bufferSize)
 	}
```

Comment:
We remove this check and expect the user to set reasonable defaults. This check was a problem in tests where we are switching writes in one shard and then the next. With this check, the second time we wouldn't buffer because the check forced a min time between buffering to be too high.